src/tests/actions/socialLogin/GoogleActions.test.js

```diff
@@ -0,0 +1,69 @@
+import configureMockStore from 'redux-mock-store';
+import thunk from 'redux-thunk';
+import moxios from 'moxios';
+
+import {
+  GOOGLE_AUTHENTICATION,
+  GOOGLE_AUTH_SUCCESS,
+  GOOGLE_AUTH_FAIL,
+} from '../../../actions/socialLoginTypes';
+
+import LoginGoogle from '../../../actions/socialLogin/GoogleActions';
+import data from '../../mockData';
+
+const middlewares = [thunk];
+const mockStore = configureMockStore(middlewares);
+
+describe('Google authentication process', () => {
+  beforeEach(() => {
+    moxios.install();
+  });
+
+  afterEach(() => {
+    moxios.uninstall();
+  });
+
+  it('logs in successfully', () => {
+    const store = mockStore({});
+    moxios.wait(() => {
+      const request = moxios.requests.mostRecent();
+      request.respondWith({
+        status: 200,
+        response: data.soicalAuthData.success,
+      });
+      document.location.href = '/successpage';
+    });
```

Comment:
For redirecting I think you should stick to history.push() instead of document.location.href